build.py

```diff
@@ -213,6 +213,12 @@ def run_tests(race, parallel, timeout, no_vet):
             return False
     else:
         logging.info("Skipping 'go vet' call...")
+
+    # NOTE: TestPipelineImplemented in pipeline/tick/tick_test.go uses the deprecated importer.For to load
+    # packages from source, requiring either GOPATH to be set or deps to be vendored.
```

Comment:
I have no idea how tests passed without this before. I see a few merges to our cross-builder image definition since the last merge to `master` here, but none of the x-builder changes look like they'd have broken things around GOPATH or vendoring deps.

Reply:
Are we pinning the cross-builder image? If we're using 'latest' anywhere that could explain some of the weirdness here - if so we should pin it.

Reply:
I think all the x-builder changes made to get plutonium building are/will be needed for kapacitor too, but I'll swap `latest` with the hash of the latest build.